docs/content/en/docs/reference/support/support-scope.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Support scope"
+linkTitle: "Support scope"
+weight: 50
+description: >
+  Support scope for EKS Anywhere
+---
+
+Enterprise support for Amazon EKS Anywhere is available to Amazon customers who pay for the [Amazon EKS Anywhere Enterprise subscription.](https://aws.amazon.com/eks/eks-anywhere/pricing/)
+If you would like to purchase the Amazon EKS Anywhere Enterprise Subscription, contact an [AWS specialist.](https://aws.amazon.com/contact-us/sales-support-eks/) 
+
+EKS Anywhere is an open source project and it is supported by the community.
+If you have a problem, open an [issue](https://github.com/aws/eks-anywhere/issues) and someone will get back to you as soon as possible.
+If you discover a potential security issue in this project, we ask that you notify AWS/Amazon Security via our [vulnerability reporting page.](http://aws.amazon.com/security/vulnerability-reporting/)
+Please do not create a public GitHub issue for security problems.
+
+## Operating system support
+
+EKS Anywhere has some level of support for the following operating system nodes:
+
+* **Bottlerocket**: Bottlerocket is the only fully supported operating system for EKS Anywhere nodes.
+Bottlerocket OVAs and images are distributed by the EKS Anywhere project.
+See the [Artifacts]({{< relref "../artifacts" >}}) page for details.
+
```

Comment:
Need to add rhel here?

Reply:
Yes, @chrisnegus

Reply:
Okay. I added a section on support for RHEL-based images.